.solcover.js

```diff
@@ -1,3 +1,16 @@
+const { execSync } = require("child_process");
+const log = console.log;
+
+// Copies pre-built token artifacts to .coverage_artifacts/contracts
+function provisionTokenContracts(config){
```

Comment:
This token provisioning is now done in a post-compilation hook executed by the Truffle plugin. You're also usually compiling the DappSys contracts during this step but Truffle kept reporting there was nothing to compile there, so I just skipped that. But if it's necessary it would go in this function...